frontend/ui/elements/common_info_menu_table.lua

```diff
@@ -49,5 +51,32 @@ common_info.report_bug = {
         })
     end
 }
+if Device:isKobo() then
+    common_info.reboot = {
+        text = _("Reboot"),
+    	  callback = function()
```

Comment:
Also there's something odd about the indentation here. 4 spaces followed by a tab and then a space? Just stick with 4 spaces per indentation. :-)